Task 2/Get The Middle Character.js

```diff
@@ -0,0 +1,10 @@
+/*
+Question Link
+http://www.codewars.com/kata/get-the-middle-character
+*/
+function getMiddle(myString) {
```

Comment:
The parameter `myString` can be renamed to a more meaningful name. Something like `inputString`.